src/NServiceBus.Core/Transports/Msmq/MsmqMessageSender.cs

```diff
@@ -50,7 +51,9 @@ public void Send(TransportMessage message, Address address)
                                 new MessageQueue(MsmqUtilities.GetReturnAddress(message.ReplyToAddress.ToString(),
                                                                                 address.ToString()));
 
-                        if (UnitOfWork.HasActiveTransaction())
+                        bool suppressNativeTransactions;
```

Comment:
>is there a situation when we want to allow for native transactions other than when dtc is enabled?

Native transactions is only in use when you're not using the DTC, can you clarify the question?